ui/src/styles/styles.scss

```diff
@@ -1,4 +1,4 @@
-/*!
+/*
```

Comment:
These CSS changes are for more than just the React stuff, right?

Reply:
Yes. Lmk if you see anything that's changed with the angular version. I hadn't noticed anything.